go.mod

```diff
@@ -6,8 +6,14 @@ require (
 	github.com/go-chi/chi v4.1.2+incompatible
 	github.com/go-chi/cors v1.1.1
 	github.com/go-chi/render v1.0.1
+	github.com/gorilla/mux v1.8.0
```

Comment:
these dependencies seem unnecessary. A good tip is to do:
```
go mod tidy
```
This will make sure dependencies are up to date and removes unnecessary ones